integration/spring-boot/autoconfigure/pom.xml

```diff
@@ -74,6 +74,16 @@
             <artifactId>spring-boot-autoconfigure-processor</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
```

Comment:
I think this dependency is not need, or there are reasons to include it?

Reply:
I'll check it, but probably yes, good catch!